cpu/stm32_common/periph/Makefile

```diff
@@ -15,4 +15,8 @@ ifneq (,$(filter periph_flashpage periph_eeprom,$(USEMODULE)))
   SRC += flash_common.c
 endif
 
+ifneq (,$(filter periph_stm32_eth,$(FEATURES_PROVIDED)))
+  SRC += eth.c
```

Comment:
How to include the pheriph driver automatically when FEATURES_REQUIRED is set? I dont this this is the correct way

Reply:
Removed